tests/zfs-tests/cmd/Makefile.am

```diff
@@ -134,3 +134,8 @@ scripts_zfs_tests_bin_PROGRAMS += %D%/randfree_file
 scripts_zfs_tests_bin_PROGRAMS += %D%/file_fadvise
 %C%_file_fadvise_SOURCES  = %D%/file/file_fadvise.c
 endif
+
+if BUILD_FREEBSD
+scripts_zfs_tests_bin_PROGRAMS += %D%/file_fadvise
+%C%_file_fadvise_SOURCES  = %D%/file/file_fadvise.c
+endif
```

Comment:
Since only FreeBSD and Linux are currently supported this can be built unconditionally now.